frontend-zs/src/components/stories/StoryPage.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react'
+import { Button, Media, Container } from 'react-bootstrap'
+import Axios from 'axios'
+import { Link, useParams } from 'react-router-dom'
+import { TiArrowBack } from "react-icons/ti"
+import StoryDetails from './StoryDetails'
+import StoryPagination from './StoryPagination'
+import StoryQuestionLike from './StoryQuestionLike'
+import { authenticationService } from '../../services/authentication.service';
+import './Story.scss'
+
+function goBack() {
```

Comment:
Do we really need a back button? Can't we just rely on the browser functionalities for that?

Reply:
We can ask Felix. The mockups have a back button. Maybe it is not that intuitive for children to go back on a browser.

Reply:
Okay then, can we just align the button with the text below it?

Reply:
Sure.  

![image](https://user-images.githubusercontent.com/29261118/80300274-e33bfd80-87a3-11ea-9b23-b4961edcce68.png)